tests/kafkatest/services/mirror_maker.py

```diff
@@ -133,12 +145,19 @@ def start_node(self, node):
         node.account.ssh("mkdir -p %s" % MirrorMaker.LOG_DIR, allow_fail=False)
 
         # Create, upload one consumer config file for source cluster
-        consumer_props = self.render('consumer.properties', zookeeper_connect=self.source.zk.connect_setting())
+        if self.new_consumer:
+            consumer_props = self.render("new_consumer.properties", bootstrap_servers=self.source.bootstrap_servers())
+        else:
+            consumer_props = self.render("consumer.properties", zookeeper_connect=self.source.zk.connect_setting())
+            consumer_props += "\noffsets.storage=%s\n" % self.offsets_storage
+
         node.account.create_file(MirrorMaker.CONSUMER_CONFIG, consumer_props)
+        self.logger.info("Mirromaker consumer props:\n" + consumer_props)
```

Comment:
This doesn't seem like INFO level data -- probably more like DEBUG.

Also type -- Mirromaker -> Mirrormaker